shared-libs/transitions/src/lib/utils.js

```diff
@@ -313,5 +314,7 @@ module.exports = {
   isNonEmptyString: expr => typeof expr === 'string' && expr.trim() !== '',
   evalExpression: (expr, context) => vm.runInNewContext(expr, context),
 
-  getSubjectIds: contact => registrationUtils.getSubjectIds(contact)
+  getSubjectIds: contact => registrationUtils.getSubjectIds(contact),
+
+  XFORM_CONTENT_TYPE
```

Comment:
I think instead of exposing a string it'd be more useful to expose a function, eg:

```js
isXFormReport: doc => doc && doc.content_type === 'xml' 
```

It gives more context about how it should be used and removes some duplicate code too!

Reply:
Done!